addon/utils/ce/handlers/emphasis-markdown-handler.js

```diff
@@ -48,23 +49,23 @@ export default EmberObject.extend({
    * @return boolean
    * @public
    */
-  isHandlerFor(event) {
+  isHandlerFor(event){
     return event.type === "keydown" &&
-      this.get('rawEditor.currentSelectionIsACursor') &&
+      this.rawEditor.currentSelectionIsACursor &&
       event.key == ' ' &&
-      this.nodeContainsRelevantMarkdown(this.get('currentNode'));
-  },
+      this.nodeContainsRelevantMarkdown(this.rawEditor.currentNode);
+  }
 
   /**
-   * handle the event
+   * handle arrow event
```

Comment:
make sure to keep the comment inline with handler, in this case not handling arrow events